python/lsst/ci/prepare.py

```diff
@@ -1,11 +1,15 @@
 #############################################################################
 # Preparer
-
+from __future__ import annotations
+import asyncio
 from io import open
```

Comment:
This is a pre-existing issue, but please consider deleting this import (here and anywhere else found). `open` is a builtin. I have never seen anyone import open from the io module before.